pkg/assets/assets.go

```diff
@@ -102,13 +104,30 @@ func assetFromTemplate(name string, tb []byte, data interface{}) (*Asset, error)
 	return &Asset{Name: name, Data: bs}, nil
 }
 
-type FileInfoPredicate func(os.FileInfo) bool
+type FileInfoPredicate func(string, os.FileInfo) bool
```

Comment:
nit: naming the string as path would be helpful

Reply:
Done.